src/auth/dto/password-reset.dto.ts

```diff
@@ -0,0 +1,13 @@
+import{ IsByteLength, IsNotEmpty, MaxLength } from 'class-validator';
+import { MatchesWithProperty } from 'src/validators/match-other.validator';
+
+export class PasswordResetDto {
+    @IsNotEmpty()
+    @IsByteLength(0, 72)
+    @MaxLength(72)
+    password: string;
+
+    @IsNotEmpty()
+    @MatchesWithProperty(PasswordResetDto, o => o.password)
+    passwordVerification: string;
```

Comment:
This should implement the password regex, like the password that is set during account creation. See `create-user.dto.ts` file for reference

```ts
@Matches(PASSWORD_REGEX, {
        message: 'Password should be minimum eight character long and have at least one uppercase letter, one lowercase letter, one number and one special character.'
    })
```